app/utils/disk-store.ts

```diff
@@ -24,27 +24,27 @@ interface LedgerWallet {
 export type DiskStore = LedgerWallet | SoftwareWallet;
 
 export const persistEncryptedMnemonic = (encryptedMnemonic: string) => {
-  void api.store.set(StoreIndex.EncryptedMnemonic, encryptedMnemonic);
+  void main.store.set(StoreIndex.EncryptedMnemonic, encryptedMnemonic);
```

Comment:
why are these promises ignored?

Reply:
No need to block UI?

Reply:
Good spot, it was like this pre-using preload.js set up, fixed.